lib/mongoid/scroll/base_cursor.rb

```diff
@@ -1,7 +1,7 @@
 module Mongoid
   module Scroll
     class BaseCursor
-      attr_accessor :value, :tiebreak_id, :field_type, :field_name, :direction, :include_current
+      attr_accessor :value, :tiebreak_id, :field_type, :field_name, :direction, :include_current, :previous
```

Comment:
I'm not sure about the naming of the previous flag.

Reply:
Per my comment in https://github.com/mongoid/mongoid-scroll/pull/38#issuecomment-2311028872, this could be the cursor itself, `previous_cursor` and it would be deserialized into a `BaseCursor`.